summit-test/setup_repos

```diff
@@ -1,9 +1,19 @@
 #!/bin/bash
-cd /
-mkdir deploy-lsstts
-cd deploy-lsstts
-mkdir docker_tmp
-chmod 777 docker_tmp
-git clone https://github.com/lsst-it/docker-compose-ops.git
-git clone https://github.com/lsst-ts/ts_ddsconfig.git
-cd ~/
+main_dir=deploy-lsstts
+docker_dir=docker_tmp
+#cd /
+if [ ! -e ${main_dir} ]; then
+  mkdir ${main_dir}
+fi
+cd ${main_dir}
+if [ ! -e ${docker_dir} ]; then
+  mkdir ${docker_dir}
+  chmod 777 ${docker_dir}
+fi
+if [ ! -e docker-compose-ops ]; then
+  git clone https://github.com/lsst-it/docker-compose-ops.git
+fi
```

Comment:
Wouldn't you want to do a `docker pull` or `docker fetch` if the directory already exists? Same for `ts_ddsconfig` below.

Reply:
I think you mean git pull etc., but in this case that's what the update_repo script handles. This script is only mean for first time deployment.

Reply:
Sorry, `git pull` yes. OK thanks for the explanation.

Reply:
Since this is supposed to be a one-off setup script, maybe you want to add an else statement in line 8, add an error message explaining it and exit with code 1.

Also, can you add headers with comments explaining what this script is made for?